api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -51,7 +52,7 @@ public CohortMembership(Integer patientId) {
 	public boolean isActive(Date asOfDate) {
 		Date date = asOfDate == null ? new Date() : asOfDate;
 		return !this.getVoided() && OpenmrsUtil.compare(startDate, date) <= 0
-				&& OpenmrsUtil.compareWithNullAsLatest(date, endDate) <= 0;
+			&& OpenmrsUtil.compareWithNullAsLatest(date, endDate) <= 0;
```

Comment:
Is this just a whitespace change?

Reply:
Yes I think so. Not sure how this happened

Reply:
Are able to revert most of the whitespace changes to improve our PR?

Reply:
Done as requested. But I am not able to remove that space easily